app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__LevelEditor.re

```diff
@@ -203,4 +203,4 @@ let make =
       </div>
     </div>
   </div>;
-};
+};
```

Comment:
Please configure your editor to [not strip trailing newlines](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). Once again, you can [check out my config](https://github.com/harigopal/config/blob/master/vscode.json) for an example.